mni_atlas_reader/atlas_reader.py

```diff
@@ -24,35 +64,31 @@ def get_vox_coord(affine, coord):
         x, y, z array index of voxel
     """
     inverse = np.linalg.inv(affine)
-    voxCoord = np.dot(inverse, np.hstack((coord, 1)))[:3]
-    return voxCoord.round().astype('int').tolist()
+    vox_coord = np.dot(inverse, np.hstack((coord, 1)))[:3]
+    return vox_coord.round().astype('int').tolist()
 
 
-def get_label(atlastype, labelID):
-    """
-    Reads out the name of a specific label
+def get_label(atlas, label_id):
+    """Reads out the name of a specific label
 
     Parameters
     ----------
-    atlastype : str
+    atlas : str
         Name of atlas to use
-    labelID : int
+    label_id : int
         Numeric label of cluster
 
     Returns
     ------
     label : str
         Atlas label name
     """
-    if 'freesurfer' in atlastype:
-        atlastype = 'freesurfer'
-    labels = np.array(pd.read_csv('atlases/labels_%s.csv' % atlastype))
-    labelIdx = labels[:, 0] == labelID
-    if labelIdx.sum() == 0:
-        label = 'No_label'
-    else:
-        label = labels[labelIdx][0][1]
-    return label
+
+    labels = get_atlas(atlas).labels
+    try:
+        return labels.query('index=={}'.format(label_id)).name.iloc[0]
```

Comment:
Request to learn: How does `sklearn`'s `Bunch` and the query work? Could you not have a simple list for this?

Reply:
`Bunch` is just a dictionary where you can access the keys as attributes! The `query` here is referring to the `DataFrame` stored in `labels`.